programs/programs/urgent_needs/ma/community_support_line.py

```diff
@@ -0,0 +1,15 @@
+from ..base import UrgentNeedFunction
+
+
+class CommunitySupportLine(UrgentNeedFunction):
+    dependencies = ["age"]
+    max_age = 21
```

Comment:
If a child is 21 and 11 months, we want the household to be eligible. Does the current logic support that? Seems like it should, but I want to be sure!

Reply:
Awesome, thanks for clarifying!